python/setup.py

```diff
@@ -47,7 +47,7 @@ def read(fname):
     download_url="https://github.com/logicalclocks/feature-store-api/releases/tag/"
     + __version__,
     packages=find_packages(),
-    long_description=read("README.md"),
+    long_description=read("../README.md"),
```

Comment:
```suggestion
    long_description=read("README.md"),
```
this was correct, for release branches it's different. please undo this change :)